ui/__init__.py

```diff
@@ -98,6 +132,7 @@ def _update(self):
         """
         # STUB(agent): async agent move requests may be called from here
         self._view.update()
+        self._view_dispatcher.dispatch()
```

Comment:
Dispatches all queued events onto the update frame.
Using this to dispatch the timer update events to the update frame without forcing rendering updates without other model changes.